src/web/components/StarRating.tsx

```diff
@@ -2,49 +2,51 @@ import React from 'react';
 import { css, cx } from 'emotion';
 import Star from '@frontend/static/icons/star.svg';
 import { palette } from '@guardian/src-foundations';
-import { from } from '@guardian/src-foundations/mq';
 
 const ratingsWrapper = css`
     background-color: ${palette.brandYellow.main};
     display: inline-block;
 `;
 
-const largeSize = css`
-    padding: 6px 10px;
-    svg {
-        width: 20px;
-        height: 20px;
-    }
-`;
-
-const smallSize = css`
-    padding: 2px;
-    margin-bottom: 5px;
-    margin-top: 5px;
-    svg {
-        width: 15px;
-        height: 15px;
-    }
-
-    ${from.wide} {
-        padding: 2px;
-        svg {
-            width: 23px;
-            height: 23px;
-        }
-    }
-`;
+type SizeType = 'large' | 'medium' | 'small';
 
 const emptyStar = css`
     fill: transparent;
     stroke: ${palette.neutral[7]};
 `;
 
+const determinSize = (size: SizeType) => {
+    switch (size) {
+        case 'small':
+            return css`
+                padding: 3px;
+                svg {
+                    width: 12px;
+                    height: 12px;
+                }
+            `;
+        case 'medium':
+            return css`
+                padding: 4px;
+                svg {
+                    width: 14px;
+                    height: 14px;
+                }
+            `;
+        case 'large':
+            return css`
+                padding: 2px;
+                svg {
+                    width: 20px;
+                    height: 20px;
+                }
+            `;
+    }
+};
 export const StarRating: React.FC<{
     rating: number;
-    size: string;
-}> = ({ rating, size }) => {
-    const sizeClass = size === 'largeSize' ? largeSize : smallSize;
+    size: SizeType;
+}> = ({ rating, size = 'small' }) => {
```

Comment:
I think this is my mistake but if `size` is required there's no need to set a default